charts/rancher-gke-operator/105.0.0+up1.10.0-rc.2/Chart.yaml

```diff
@@ -2,12 +2,12 @@ annotations:
   catalog.cattle.io/auto-install: rancher-gke-operator-crd=match
   catalog.cattle.io/certified: rancher
   catalog.cattle.io/hidden: "true"
-  catalog.cattle.io/kube-version: '>= 1.28.0-0 < 1.31.0-0'
+  catalog.cattle.io/kube-version: '>= 1.28.0-0 < 1.32.0-0'
   catalog.cattle.io/namespace: cattle-system
   catalog.cattle.io/os: linux
   catalog.cattle.io/permits-os: linux,windows
   catalog.cattle.io/provides-gvr: gkeclusterconfigs.gke.cattle.io/v1
-  catalog.cattle.io/rancher-version: '>= 2.9.0-0 < 2.10.0-0'
+  catalog.cattle.io/rancher-version: '>= 2.9.0-0 < 2.11.0-0'
```

Comment:
Shouldn't this be >= 2.10.0-0 ?

Reply:
We'll keep this as is to keep it consistent with other operators that have been updated already.